schedule/yast/maintenance/create_hdd_transactional_server_restapi.yaml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
instead of using `_restapi` let's move to a folder `/maintenance/test_cases/create_hdd_transactional_server.yaml`. Inside we will have all the files that are common for both products.

Reply:
It makes sense to oragnize things, but it will be hard to understand why some schedules are in a sub-folder and some not, for someone who does not know the context. I would suggest at least to add a "README" file. The suffix _restapi has the advantage to be immediately understandable.

Reply:
yaml based on default can work without libyui-rest-api, there is a mix of technologies, it is never a good idea to call things with name not related with funcitonal or end-to-end testing and use internals to do it.
I cannot see the future solution yet until we iterate with this a bit more, but the idea is to have the test cases well located. The plan probably is to move everything else that cannot be refactor this way to some folder with 'some' name. What I try to prevent is to use something specific like 'libyui-rest-api' or 'new-thingy' when actually should be our default way in a few months I hope.
This could be a possible structure:
- yam/
  - tc1
  - tc2
  - tc3
  - tc4/ (when exceptionally we need more than 1 file)
  - the_rest/
  - defaults/
But I'm trying not to ask to move all old stuff now to folder because that would create too many failure likely, so moving the new makes more sense. wdyt @JRivrain ?

Reply:
I agree with the idea of  sub-folder, but it would be good to have something explaining why some test suites are using restapi, and some not. Maybe a comment in the old yamls, like "this yaml is legacy" with explaination that prior to sp4 we don't have restapi. Otherwise it will be hard to understand.

Reply:
let's leave it like that for now with `_restapi`, I think all will make sense later.